### ISSUE TITLE
4K Support

### ISSUE THREAD

**Issue by DragonWario** [Moehammered/switch-remote-play#47]
Could this be updated to support 4K monitors sometime? 1440p leaves out a significant portion of my screen.

**Comment by Moehammered** [comment]
@DragonWario sure thing I'll go ahead and add it when I resume work on this project in the next month or so.

When you say it leaves a portion of the screen out, doesn't your screen resolution get changed to 1440p? Or does the monitor remain at 4K and the switch only captures a 1440p region?

The program usually changes the resolution of the monitor to match your chosen desktop resolution in the application. I wonder what's going on there for you.

**Comment by DragonWario** [comment]
Yeah, Resolution stays at 4K for me for some reason.. Result is only a portion of of the screen being streamed. 
It may have something to do with how I use two DisplayPort cables (4K@144Hz is too much bandwidth for just one cable).

**Comment by Moehammered** [comment]
@DragonWario the DisplayPort I'd hope wouldn't have an affect on it. Regardless, I did a quick modification and made a version of the Switch app that will allow for selecting 4K resolution.
Please test the attached version of the switch app (just overwrite the existing one at switch/switch-remote-play) and see if it works for you.
[switch-remote-play-v1.0.1-4k-res.zip](https://github.com/Moehammered/switch-remote-play/files/7969965/switch-remote-play-v1.0.1-4k-res.zip)

**Comment by DragonWario** [comment]
I apologize for the late response, but it seems to work fine now. There's some lag, but that's to be expected without things like HW acceleration and only relying on the CPU.
720p30 stream, 15260KB/s, drop time Vsync, ultra fast preset, CBR were the settings I tested with.

**Comment by Moehammered** [comment]
@DragonWario Thanks that's great news! I'll integrate the fix then in the next update I make to the app. I'll probably be resuming work on the app after next month.

I appreciate your help with fixing the issue. :)
